donate/settings.py

```diff
@@ -581,3 +593,45 @@
 # Pontoon check slack bot
 SLACK_WEBHOOK_PONTOON = env('SLACK_WEBHOOK_PONTOON')
 TRAVIS_LOGS_URL = env('TRAVIS_JOB_WEB_URL', default='')
+
+# Mozilla OpenID Connect/Auth0 configuration
+
+# disable user creating during authentication
+OIDC_CREATE_USER = False
+
+# How frequently do we check with the provider that the user still exists.
+OIDC_RENEW_ID_TOKEN_EXPIRY_SECONDS = 15 * 60
+
+OIDC_RP_SIGN_ALGO = "RS256"
+OIDC_RP_CLIENT_ID = env("OIDC_RP_CLIENT_ID")
+OIDC_RP_CLIENT_SECRET = env("OIDC_RP_CLIENT_SECRET")
+
+# These values should be overwritten by thunderbird/other instances:
```

Comment:
... or.... should they?

Reply:
Can we overwrite the below strings in Thunderbird when they're not being pulled from env vars?

@cadecairos This is going to mean another refactor of django configurations, isn't it. :-|

Reply:
no idea, but hey let's start even earlier than that: _do thunderbird folks need access or not_ because that's been an unanswered question for like three weeks now. If not, then we literally don't care >_>